src/ActivityStickyQueues/FileProcessingWorkflow.cs

```diff
@@ -0,0 +1,71 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+namespace TemporalioSamples.ActivityStickyQueues;
+
+[Workflow]
+public class FileProcessingWorkflow
+{
+    [WorkflowRun]
+    public async Task RunAsync(int maxAttempts = 5)
+    {
+        for (var attempt = 1; attempt <= maxAttempts; ++attempt)
+        {
+            try
+            {
+                var uniqueWorkerTaskQueue = await Workflow.ExecuteActivityAsync(
+                    (NonStickyActivities act) => act.GetUniqueTaskQueueAsync(),
+                    new()
+                    {
+                        StartToCloseTimeout = TimeSpan.FromMinutes(1),
+                    });
+
+                var downloadPath = Path.GetTempFileName();
+
+                await Workflow.ExecuteActivityAsync(
+                    (StickyActivities act) => act.DownloadFileToWorkerFileSystemAsync("https://temporal.io", downloadPath),
```

Comment:
Usually we recommend a `record` for two or more parameters (or even single ones sometimes too) for future proofing when you want to add fields